blocks/procedures.js

```diff
@@ -783,8 +784,8 @@ Blockly.Blocks['procedures_callnoreturn'] = {
    * @this {Blockly.Block}
    */
   updateShape_: function() {
-    for (var i = 0; i < this.arguments_.length; i++) {
-      var field = this.getField('ARGNAME' + i);
+    for (let i = 0; i < this.arguments_.length; i++) {
+      let field = this.getField('ARGNAME' + i);
```

Comment:
I think this should be `const`, because the "mutation" of `field` below appears to just be variable reuse.